bin/gollum-migrate-tags

```diff
@@ -89,6 +87,8 @@ rescue OptionParser::InvalidOption
   exit
 end
 
+REPO = ARGV[0] || Dir.pwd
```

Comment:
Doing this *after* optparser parses the options means that `ARGV[0]` is whatever argument *remains* that is not recognized by optparse.